themes/default/content/docs/guides/self-hosted/_index.md

```diff
@@ -8,9 +8,7 @@ meta_desc: Pulumi Business Critical Edition gives you the option to self-host Pu
 ---
 
 {{% notes type="info" %}}
-Self-hosting is only available with **Pulumi Business Critical**. If you would like to evaluate the Self-Hosted Pulumi Service, sign up for the [30 day trial]({{< relref "/product/self-hosted#self-hosted-trial" >}}) or [contact us]({{< relref "/contact.md" >}}).
-
-To manage your state with a self-*managed* backend, such as a cloud storage bucket, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
+Self-hosting is available as part of **Pulumi Business Critical** and open source, self-managed backends. If you would like to evaluate the Self-Hosted Pulumi Service, sign up for the [30 day trial]({{< relref "/product/self-hosted#self-hosted-trial" >}}) or [contact us]({{< relref "/contact.md" >}}). To use a open source, self-managed backend, such as a cloud storage bucket, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```

Comment:
```suggestion
Self-hosting the Pulumi Service is available as part of **Pulumi Business Critical**. Pulumi also lets you manage state yourself using a self-managed backend. If you would like to evaluate the Self-Hosted Pulumi Service, sign up for the [30 day trial]({{< relref "/product/self-hosted#self-hosted-trial" >}}) or [contact us]({{< relref "/contact.md" >}}). To use an open-sourced, self-managed backend, such as a cloud storage bucket, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```

Reply:
thx for the review!
im worried that we have lost the clarity i was going for with your changes to the first sentence.
also im worried that `To use an open-sourced, self-managed backend` in the last sentence makes it seem like they have to open source their usage of it.
lets sync up about this next week!

Reply:
I would drop the second sentence because it's confusing to interject the self-managed in between the two sentences about self-hosting. Also agree with susan on not using "open-sourced". perhaps we could say "To use Pulumi open source with your own self-managed backend, such as a cloud storage bucket, ...."